doc/maintenance.rst

```diff
@@ -61,17 +61,18 @@ this, good first issues do not require a separate effort label.
 
 Some other labels can be used to describe further the topic of the issue:
 
-	- |API| This issue is related to the Nilearn's API.
-	- |Code quality| This issue tackles code quality (code refactoring, PEP8...).
-	- |Datasets| This issue is related to datasets or the :mod:`nilearn.datasets` module.
-	- |Discussion| This issue is used to hold a general discussion on a specific topic where community feedback is desired (no need to specify effort, priority, or impact here).
-	- |GLM| This issue is related to the :mod:`nilearn.glm` module.
-	- |Infrastructure| This issue describes a problem with the project's infrastructure (CI/CD...).
-	- |Installation| The issue describes a problem with the installation of Nilearn.
-	- |Maintenance| This issue is related to maintenance work.
-	- |Plotting| The issue is related to plotting functionalities.
-	- |Testing| The issue is related to testing.
-	- |Usage| This issue is a usage question and should have been posted on :neurostars:`neurostars <>`.
+-   |API| This issue is related to the Nilearn's API.
+-   |Code quality| This issue tackles code quality (code refactoring, PEP8...).
+-   |Datasets| This issue is related to datasets or the :mod:`nilearn.datasets` module.
+-   |Discussion| This issue is used to hold a general discussion on a specific topic
+    where community feedback is desired (no need to specify effort, priority, or impact here).
+-   |GLM| This issue is related to the :mod:`nilearn.glm` module.
+-   |Infrastructure| This issue describes a problem with the project's infrastructure (CI/CD...).
+-   |Installation| The issue describes a problem with the installation of Nilearn.
+-   |Maintenance| This issue is related to maintenance work.
+-   |Plotting| The issue is related to plotting functionalities.
+-   |Testing| The issue is related to testing.
+-   |Usage| This issue is a usage question and should have been posted on :neurostars:`neurostars <>`.
```

Comment:
These labels aren't currently rendering in the output : https://output.circle-artifacts.com/output/job/d95b1114-a721-449c-9cc1-b93b8c971245/artifacts/0/dev/maintenance.html

Reply:
weirdly, they loaded after I commented this ! not sure why the long delay -- resolving !